specification/quota/resource-manager/Microsoft.Quota/preview/2023-06-01-preview/groupquota.json

```diff
@@ -0,0 +1,1797 @@
+        "x-ms-examples": {
+          "GroupQuotaLimits-Requests_Get": {
+            "$ref": "./examples/GroupQuotaLimits-Requests/GroupQuotaLimits-Requests_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "#/parameters/RequestIdInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success, with the details of the groupQuota request.",
+            "schema": {
+              "$ref": "#/definitions/SubmittedResourceRequestStatus"
+            }
+          },
+          "default": {
+            "description": "Error",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Management/managementGroups/{mgId}/subscriptions/{subscriptionId}/{resourceProviderScope}/providers/Microsoft.Quota/groupQuotas/{groupQuotaName}/quotaAllocations": {
+      "get": {
+        "tags": [
+          "SubscriptionQuota-Allocation"
+        ],
+        "summary": "Gets all the quota assigned to a subscription for the specific Resource Provider, Location.",
+        "description": "Gets all the quota assigned to a subscription for the specific Resource Provider, Location. This will include the GroupQuota and total quota assigned to the subscription. Only the Group quota assigned to the subscription can be assigned back to the MG Group Quota.",
+        "operationId": "GroupQuotaSubscriptionQuotaAllocation_List",
+        "x-ms-examples": {
+          "SubscriptionQuota-Allocation_List_Request_ForCompute": {
+            "$ref": "./examples/SubscriptionQuota-Allocation/SubscriptionQuota-Allocation_List_Request-Compute.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/MgIdInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ScopeInParameters"
+          },
+          {
+            "$ref": "#/parameters/GroupQuotaNameInParameters"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SubscriptionQuotaDetailsList"
```

Comment:
>SubscriptionQuotaDetailsList [](http://example.com/codeflow?start=37&length=28)

Shouldn't this be named as SubscriptionGroupQuotaAllocationsList?

Reply:
updated to `SubscriptionQuotaAllocationsList` and non list GET to `SubscriptionQuotaAllocations`